csharp/examples/FluentBuilderExample/FluentBuilderExample.csproj

```diff
@@ -0,0 +1,15 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  
+  <Import Project="../../build/Common.props" />
+
+  <PropertyGroup>
+    <LangVersion>7.2</LangVersion>
```

Comment:
What do you think about putting `LangVersion` in the `Common.props`?

Reply:
I think that's a good plan, but I think I will remove Common.props and replace it with the Directory.Build.props feature.

Reply:
> replace it with the Directory.Build.props feature.

Even better!